### PR TITLE
Adding a flag for AKS to AMLARC migration and set up corresponding FE…

### PR DESCRIPTION
**Pull request by tianchijushi**
… helm values

---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [x] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [x] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your PR is merged into master branch, a new PR will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repo and upgrade the version in the PR but do not modify `src/index.json`.

### CODE REVIEW

**Review by jonathan-innis — DISMISSED** [review]
LGTM 🚀

**Review by krabhishek8260 — APPROVED** [review]

File: src/k8s-extension/azext_k8s_extension/partner_extensions/AzureMLKubernetes.py [hunk 1]
```diff
@@ -211,6 +214,12 @@ def __validate_scoring_fe_settings(self, configuration_settings, configuration_p
             configuration_settings['clusterPurpose'] = 'DevTest'
         else:
             configuration_settings['clusterPurpose'] = 'FastProd'
+        isAKSMigration = _get_value_from_config_protected_config(
+            self.IS_AKS_MIGRATION, configuration_settings, configuration_protected_settings)
+        isAKSMigration = str(isAKSMigration).lower() == 'true'
+        if isAKSMigration:
+            configuration_settings['scoringFe.namespace'] = "default"
+            configuration_settings['scoringFe.release-name'] = "scoring-fe-migration"
```
[the remark this answers is not in the document]
Comment (tianchijushi) [reply src/k8s-extension/azext_k8s_extension/partner_extensions/AzureMLKubernetes.py]: Agree. I think you are looking at an old pr version. In the latest version we skip setting up the release-name now and then in AMLARC we could use Helm Release.name instead

**Review by jonathan-innis — APPROVED** [review]
🚀

**Pull request merged**